developer-docs-site/docs/move/book/variables.md

```diff
@@ -330,18 +330,23 @@ let (x1, y, z1) = three(); // ERROR!
 All of the different structures in `let` can be combined! With that we arrive at this general
 grammar for `let` statements:
 
-> _let-binding_ → **let** _pattern-or-list_ _type-annotation_<sub>_opt_</sub>
-> _initializer_<sub>_opt_</sub> > _pattern-or-list_ → _pattern_ | **(** _pattern-list_ **)** >
-> _pattern-list_ → _pattern_ **,**<sub>_opt_</sub> | _pattern_ **,** _pattern-list_ >
+> _let-binding_ → **let** _pattern-or-list_ _type-annotation_<sub>_opt_</sub> _initializer_<sub>_opt_</sub>
+
+> _pattern-or-list_ → _pattern_ | **(** _pattern-list_ **)**
+
+> _pattern-list_ → _pattern_ **,**<sub>_opt_</sub> | _pattern_ **,** _pattern-list_
+
 > _type-annotation_ → **:** _type_ _initializer_ → **=** _expression_
```

Comment:
The initializer clause also should go on his own line.

Reply:
done